### ISSUE TITLE
[Bug?] "Make Default"-Polygon for CompositeItem

### ISSUE THREAD

**Issue by skusku** [UnderwaterApps/overlap2d#345]
Using Make-Default Polygon on composite Items does not return an actual AABB.
Rather it returns a [0, 0, width, height] Box, completely ignoring the lower left part of the object.

**Comment by skusku** [comment]
I am working on this problem

**Comment by azakhary** [comment]
oh. got it. What is the consequence of this by the way?

**Comment by skusku** [comment]
How i found it out: I had an object that was 62x62 pixels but i wanted it to fit into a grid of 64x64, leaving 1 pixel gap around it free. I used a composite item to put my 62x62 object inside, and moved the object inside by 1 pixel² so that I could drag and drop the Composite item into the scene without having to fiddle around with the spacing. I wanted to use the Composite (1) as a library item for fast drag and drop and (2) as a "Positional Wrapper"

I wanted 1 polygon in the parent object so that I had 1 fixture in the body, but it should fit the images that were inside the CompositeItem

The Make-Default button however, instead of giving me the expected 62x62 polygon, moved by 1 px² gave me a 63x63 polygon starting from 0,0

Does that make sense?

**Comment by azakhary** [comment]
Got it! Yeah that has to be fixed. Never had to do it that precise before :))

**Comment by skusku** [comment]
Is there a built-in way to init boundingBoxes for non-composite items or would I have to implement that myself - say - in EntityUtils ?

**Comment by azakhary** [comment]
you mean just default bounding box?

**Comment by azakhary** [comment]
check this out: https://github.com/UnderwaterApps/overlap2d/blob/master/overlap2d/src/com/uwsoft/editor/view/ui/properties/panels/UIPolygonComponentPropertiesMediator.java#L105

**Comment by skusku** [comment]
Yep, working on that. And I think I got the problem by it's root :P

**Comment by azakhary** [comment]
Google luck, thanks for helping us out! :)

**Comment by skusku** [comment]
Got it! Weeeeee :)

**Comment by skusku** [comment]
Only problem : I don't know how to push the fix :D

**Comment by azakhary** [comment]
well, what step are you stuck on?

**Comment by azakhary** [comment]
The general flow is, you fork the repo, you clone your fork, you do some fixes on your clone, you commit them, you push them to your fork, then you make a pull request.

**Comment by skusku** [comment]
Okay. I skipped the forking part.
